<launch>
  <node name='cf_supervisor' pkg='drone_arena' type='cf_supervisor.py' output="screen">
    <rosparam param="crazyradio" command="load" file="$(find drone_arena)/config/crazyradio.yaml"/>
  </node>

</launch>
